.eslintrc

```diff
@@ -23,7 +24,8 @@
         "ignores": []
       }],
       "vue/singleline-html-element-content-newline": "off",
-      "no-console": ["warn", {"allow": ["warn", "error"]}]
+      "vue/no-v-for-template-key": "off",
+      "vue/no-v-for-template-key-on-child": "error"
```

Comment:
was genau soll das (machen)?

Reply:
In die richtung von v-for bin ich doch nicht gegangen. Aber das enforced die v3 version, von dem `<transition> `element. Da war der linter einfach falsch.
https://eslint.vuejs.org/rules/no-v-for-template-key-on-child.html

Reply:
ich hab keine ahnung, was das ganze heißt